faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Datatype Tabular
+area: Getting inputs right    # FAQs will be grouped by these areas on the FAQ page
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+
+---
+ 
+### Help for Tabular Datasets
```

Comment:
you don't need a title here, the title defined in the metadata above will be placed at the top of the FAQ